code/_helpers/text.dm

```diff
@@ -319,6 +319,25 @@
 /proc/trim(text)
 	return trim_left(trim_right(text))
 
+// Remove starting and ending <br> and variants
+/proc/trim_trailing_linebreaks(var/text)
```

Comment:
This should probably be replaced with the regex solution, if only to only do one pass through the string.